src/views/SalesAdd.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <v-container>
+    <v-sheet width="300" class="mx-auto">
+      <div class="mx-auto text-center" style="font-size: 24px">
+        {{ pageTitle }}
+      </div>
+      <v-form @submit.prevent="handleSubmit" ref="form">
+        <v-select
+          label="Seller"
+          v-model="seller"
+          :items="allUsersNames"
+          :rules="[validateAuthors(seller)]"
+        >
+        </v-select>
+
+        <v-select
+          v-model="buyer"
+          label="Buyer"
+          :items="allUsersNames"
+          :rules="[validateAuthors(buyer)]"
+        >
+        </v-select>
+
+        <v-container class="d-flex justify-space-between">
+          <v-btn
+            color="primary"
+            @click="toggleDialog(true)"
+            :disabled="!selectedUser"
+          >
+            Products
+          </v-btn>
+
+          <v-btn
+            color="green lighten-1"
+            @click="() => (calendarDialog = true)"
+            :disabled="!selectedUser"
```

Comment:
Do we need to disable the calendar until the seller is chosen. I believe it can be active and it shouldn't affect the rest of the functionality.

Reply:
we can, but i made it to work simultaneously as the products button to look better